### DISCUSSION TITLE
Beta test feedback from Users

### DISCUSSION THREAD

**Discussion by dennyabrain** [tattle-made/Uli#12] [General]
Lets list any feedback from the beta testers or annotators here. We can later convert them into issues and track them.

Feature request from Cheshta : 
- [ ] Allow users to resume from where they left.
If a user logs out after finishing 30 annotations, when they log back in, the session should start at post 31 and not post 1.

**Comment by tarunima** [comment]
The '&' character in the tweets is getting parsed as &amp. For example, text in the original tweet `'Since decades lacs of husbands & their entire families'` is shown in our database and UI as: `'Since decades lacs of husbands &amp; their entire families were'`
This error seems to emerge at the scraping step.

**Reply by tarunima** [reply]
@dennyabrain error still persists in updated UI.

**Reply by dennyabrain** [reply]
Since you said this had something to do with the scraping step, I did not investigate it. I just did and I can verify that this &amp is present in the .csv file that @mlkorra gave me. 
@mlkorra can you verify where this transformation of & into &amp; is happening? `&amp;` is how ampersand is written in html, so I am guessing that if the scraper is using webscraping, it might be getting this wrong.

**Comment by tarunima** [comment]
image is extending beyond the box:
<img width="507" alt="Screen Shot 2021-12-01 at 12 32 38 PM" src="https://user-images.githubusercontent.com/13188366/144187630-8b315b71-dd89-40d1-9024-28f7b346b4e3.png">

**Comment by tarunima** [comment]
None of the above option missing in last question

**Comment by tarunima** [comment]
possible for people to navigate to a post based on the post number? If I close the browser at post 50, when i restart the annotations I have to click next 50 times.

**Comment by tarunima** [comment]
Clicking on the radio buttons for this post doesn't select the option:
<img width="746" alt="Screen Shot 2021-12-01 at 12 47 28 PM" src="https://user-images.githubusercontent.com/13188366/144189034-cf6d4e9c-ab40-4e17-9634-d34ca755533f.png">
